lib/src/components/NxTransferList/NxTransferList.tsx

```diff
@@ -19,8 +20,10 @@ const defaultItemsCountFormatter = (kind: string) => (n: number) => `${n} item${
     defaultAvailableItemsCountFormatter = defaultItemsCountFormatter('available'),
     defaultSelectedItemsCountFormatter = defaultItemsCountFormatter('transferred');
 
-export default function NxTransferList<T extends string | number>(props: Props<T>) {
+export default function NxTransferList
```

Comment:
There's a lot of uses of native array methods like `.map()` and `.filter` here.  I try to be consistent about using ramda's helpers instead of those native methods, _especially_ in files that also use a bunch of other ramda functions.

Reply:
> There's a lot of uses of native array methods like `.map()` and `.filter` here. I try to be consistent about using ramda's helpers instead of those native methods, _especially_ in files that also use a bunch of other ramda functions.

Refactored here: 774e9702b6a1123bac0705354974edd6087775a5